utils/novels.js

```diff
@@ -0,0 +1,11 @@
+import fetchNovels from '../actions/novels'
```

Comment:
This would work if that function was `export default` but since it isn't what is exported is an object with the function in it so this needs to use destructuring

```JavaScript
import { fetchNovels } from '../actions/novels'
```